tools/src/launcher/Main.hs

```diff
@@ -202,6 +208,26 @@ main = do
                     , loUpdateArchive)
                     loNodeTimeoutSec
                     loReportServer
+  where
+    addConfigurationOptions :: ConfigurationOptions -> [Text] -> [Text]
```

Comment:
This is rather mda because when somebody updates `ConfigurationOptions`, they'll likely forget to update this function. I don't know a way to make GHC warn you when you haven't used all fields from a record, so I think you should do one of the following:

* `addConfigurationOptions opt@(ConfigurationOptions _ _ _) = ...`
* move `addConfigurationOptions` to the place where `ConfigurationOptions` is defined
* add a commend to the place where `ConfigurationOptions` is defined, saying “please update blah blah when you add a field here”

Reply:
Right, I will use the first way, give me a minute.